### DISCUSSION TITLE
test strategy

### DISCUSSION THREAD

**Discussion by zmrdltl** [tmccombs/hcl2json#123] [Q&A]
I want to create a `test folder` and verify whether it is normally converted to `hcl -> json` or `json -> hcl` through the go test command. Can you tell me if you have any good strategies?

**Comment by tmccombs** [comment]
I'm not sure I really understand what you are trying to do.  Are you wanting to add tests to this project, or use this tool to run tests in another project?

If the latter, I don't really know of a good strategy for that, sorry.
